### PR TITLE
Added universal propositions bank for French and German

### PR DESCRIPTION
**Pull request by Dabendorf**
I have added the German and the French data from he Universal Proposition Bank (https://github.com/System-T/UniversalPropositions)

### CODE REVIEW

File: flair/datasets/sequence_labeling.py [hunk 1]
```diff
@@ -1597,3 +1597,97 @@ def _download_wikiner(language_code: str, dataset_name: str):
...
+        :param base_path: Default is None, meaning that corpus gets auto-downloaded and loaded. You can override this
+        to point to a different folder but typically this should not be necessary.
+        :param in_memory: If True, keeps dataset in memory giving speedups in training.
+        :param document_as_sequence: If True, all sentences of a document are read into a single Sentence object
+        """
+        if type(base_path) == str:
+            base_path: Path = Path(base_path)
+
+        # column format
+        columns = {1: "text", 10: "frame"}
```
Comment (alanakbik) [inline flair/datasets/sequence_labeling.py:1621]: the column are wrong, it seems that column 10 is not the frame information

File: flair/datasets/sequence_labeling.py [hunk 2]
```diff
@@ -1597,3 +1597,97 @@ def _download_wikiner(language_code: str, dataset_name: str):
...
+        cached_path(f"{up_de_path}de-up-test.conllu", Path("datasets") / dataset_name)
+
+        super(UP_GERMAN, self).__init__(
+            data_folder,
+            columns,
+            encoding="utf-8",
+            train_file="de-up-train.conllu",
+            test_file="de-up-dev.conllu",
+            dev_file="de-up-test.conllu",
+            in_memory=in_memory,
```
Comment (alanakbik) [inline flair/datasets/sequence_labeling.py:1644]: the `comment_symbol` parameter is missing here (the UP and UD datasets have comments that should not be read)

**Review by alanakbik — CHANGES_REQUESTED** [review]
I've tested with the following code and the output does not look fully correct: 

```python
from flair.datasets import UP_GERMAN

# load corpus
corpus = UP_GERMAN()
print(corpus)

# print first sentence in train data
print(corpus.train[0])
```

This prints: 

~~~
Sentence: "sent_id Sehr gute Beratung , schnelle Behebung der Probleme , so stelle ich mir Kundenservice vor ."   [− Tokens: 17  − Token-Labels: "sent_id Sehr gute Beratung , schnelle Behebung der Probleme , so <AM-MNR> stelle ich <A0> mir Kundenservice <A1> vor ."]
~~~

Two problems here: 

(1) the sentence is read as "_sent_id Sehr gute Beratung , schnelle Behebung der Probleme , so stelle ich mir Kundenservice vor ._", but **sent_id** should not be part of the sentence. The problem is that the UP files have lines that are comments. These lines are prefixed by a # symbol and should be skipped. You can get this behavior by setting the `column_symbol` in the ColumnCorpus class.

(2) the frames are not annotated. The annotation is printed as `"_sent_id Sehr gute Beratung , schnelle Behebung der Probleme , so <AM-MNR> stelle ich <A0> mir Kundenservice <A1> vor ._"` But the annotation should be the verbs. So the object currently selects the wrong column as frame.

File: flair/datasets/sequence_labeling.py (hunk 1 above)
Comment (Dabendorf) [reply flair/datasets/sequence_labeling.py]: It was column nr. 9, starting with 0 to count, sorry, my mistake. It got fixed

File: flair/datasets/sequence_labeling.py (hunk 2 above)
Comment (Dabendorf) [reply flair/datasets/sequence_labeling.py:1644]: I have added comment_symbol="#" in both classes, thank you

File: flair/datasets/sequence_labeling.py [hunk 3]
```diff
@@ -1597,3 +1597,99 @@ def _download_wikiner(language_code: str, dataset_name: str):
...
+        cached_path(f"{up_de_path}de-up-dev.conllu", Path("datasets") / dataset_name)
+        cached_path(f"{up_de_path}de-up-test.conllu", Path("datasets") / dataset_name)
+
+        super(UP_GERMAN, self).__init__(
+            data_folder,
+            columns,
+            encoding="utf-8",
+            train_file="de-up-train.conllu",
+            test_file="de-up-dev.conllu",
+            dev_file="de-up-test.conllu",
```
Comment (alanakbik) [inline flair/datasets/sequence_labeling.py:1643]: This is switched: You are loading the dev split as `test_file` and the test split as `dev_file`.

File: flair/datasets/sequence_labeling.py [hunk 4]
```diff
@@ -1597,3 +1597,99 @@ def _download_wikiner(language_code: str, dataset_name: str):
...
+        cached_path(f"{up_fr_path}fr-up-dev.conllu", Path("datasets") / dataset_name)
+        cached_path(f"{up_fr_path}fr-up-test.conllu", Path("datasets") / dataset_name)
+
+        super(UP_FRENCH, self).__init__(
+            data_folder,
+            columns,
+            encoding="utf-8",
+            train_file="fr-up-train.conllu",
+            test_file="fr-up-dev.conllu",
+            dev_file="fr-up-test.conllu",
```
Comment (alanakbik) [inline flair/datasets/sequence_labeling.py:1691]: Same here

**Review by alanakbik — CHANGES_REQUESTED** [review]
Looks good, but test and dev splits are switched! Can you change this?

File: flair/datasets/sequence_labeling.py (hunk 3 above)
Comment (Dabendorf) [reply flair/datasets/sequence_labeling.py:1643]: Oh. Merde. Sorry for that. Such a sloppy work. I am going to fix that in a couple of minutes, thank you!

Comment (Dabendorf) [reply flair/datasets/sequence_labeling.py]: Changed it, thank you!

**Comment by alanakbik** [comment]
@Dabendorf thanks for adding this!

**Pull request merged**